src/config/config.js

```diff
@@ -5,6 +5,7 @@ const isEToro = process.env.REACT_APP_ETORO === 'true' || process.env.REACT_APP_
 
 const Config = {
   env: process.env.REACT_APP_ENV || 'development',
+  isTest: process.env.REACT_APP_ENV === 'test',
```

Comment:
remove this
any test-related things could me mocked so we don't need test-specific logic in the main source
also this approach breaks the main idea of the unit testing - we should **test components as is**, **to consider them as black boxes** (we don't know and don't need to know what's inside them)

Reply:
done